clientgui/DlgAdvPreferences.cpp

```diff
@@ -1148,6 +1153,18 @@ void CDlgAdvPreferences::OnOK(wxCommandEvent& ev) {
     ev.Skip();
 }
 
+bool CDlgAdvPreferences::ConfirmSetLocal() {
+    wxString strMessage     = wxEmptyString;
+	strMessage.Printf(
+            _("Changing to use the local BOINC preferences defined on this page. BOINC will ignore your web-based preferences, even if you subsequently make changes there. Do you want to proceed?")
```

Comment:
@CharlieFenton Does this need branding?
@RichardHaselgrove Or maybe you can find a good wording that doesn't mention BOINC?

Reply:
"Changing to use" is awkward.
Can omit "BOINC".
Can omit "defined on this page".
Could say "This will override web-based preferences".

Reply:
I've removed the need for branding, and used override instead of ignore. I'd like to keep 'defined on this page', because this whole PR is addressed at users encountering the difference between web and local preferences for the first and only time.

I haven't found a better alternative for 'Changing to use', but it's up for better suggestions.